src/Integration/Connection/MinimumSupportedSonarQubePlugin.cs

```diff
@@ -31,14 +31,16 @@ internal class MinimumSupportedSonarQubePlugin
         public static readonly MinimumSupportedSonarQubePlugin VbNet = new MinimumSupportedSonarQubePlugin(SonarPluginKeys.SonarVBNet, "SonarVB", "3.0", Language.VBNET);
 
         // Note: there is no specific technical reason for the choice of cpp v6.0 as the minimum supported version.
-        // Howver, that was the first version that uses CLang so it's close to the version embedded in SLVS
+        // However, that was the first version that uses CLang so it's close to the version embedded in SLVS
         // i.e. the analysis rules implementations should be similar so the issues shown in the IDE should be
         // similar to those reported on the server.
         public static readonly MinimumSupportedSonarQubePlugin CFamily = new MinimumSupportedSonarQubePlugin(SonarPluginKeys.SonarCFamily, "SonarCFamily", "6.0", Language.Cpp, Language.C);
 
         public static readonly MinimumSupportedSonarQubePlugin JsTs = new MinimumSupportedSonarQubePlugin(SonarPluginKeys.SonarJs, "SonarJS", "8.6", Language.Js, Language.Ts);
 
```

Comment:
About writing a comment, I checked this https://sonarcloud.io/api/plugins/installed which has this line: "version":"2.0.1 (build 611)" for the secrets plugin, and looking at the other plugins all just cut away the last bit of the versioning, so i am not sure it makes sense to write a comment just for this one and not the others.